Content.Server/StationEvents/Components/StationEventComponent.cs

```diff
@@ -87,4 +89,13 @@ public sealed partial class StationEventComponent : Component
     /// </summary>
     [DataField]
     public bool OccursDuringRoundEnd = true;
+
+    // Goobstation start
+    /// <summary>
+    ///  Expected Chaos changes when this event occurs.
+    ///  Used by the GameDirector, which picks an event expected to make the desired chaos changes.
+    /// </summary>
+    [DataField("chaos")]
+    public ChaosMetrics Chaos = new ChaosMetrics();
```

Comment:
Simplify constructor call
```suggestion
    public ChaosMetrics Chaos = new();
```